src/utils/init.ts

```diff
@@ -106,6 +95,12 @@ function init(sdkKey: string, options: InitOptions = {}): Promise<string> {
 
   initState.sdkKey = sdkKey;
   optionsAsInitState.forEach((key) => {
+    // ------------------------------------
+    // ------------------------------------
+    // I don't really get why removing authId and authClient caused this?
```

Comment:
what is the error?

Reply:
![image](https://user-images.githubusercontent.com/923485/217437159-82f9c532-ae67-4469-93b9-0cc56bdb0e71.png)

Reply:
i can't figure it out either rip, maybe leave in the expect-error without this comment?